pt/news/_posts/2016-06-20-ruby-2-4-0-preview1-released.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: news_post
+title: "Lançado Ruby 2.4.0-preview1"
+author: "naruse"
+translator: "jcserracampos"
+date: 2016-06-20 18:00:00 +0900
+lang: pt
+---
+
+Estamos satisfeitos em anunciar o lançamento do Ruby 2.4.0-preview1.
+
+Ruby 2.4.0-preview1 é o primeiro *preview* do Ruby 2.4.0.
+Este preview1 está sendo lançado antes do usual porque ele inclui tantas 
+funcionalidades novas e aprimoramentos.
```

Comment:
Based on other Portuguese posts, `improvements` has always been translated to `melhorias`, I would keep the same.